templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -237,7 +237,7 @@ if [[ "$TEST" = "azure" ]]; then
   {%- if pulp_scheme == "https" %}
   AZCERTIFI=$(/opt/az/bin/python3 -c 'import certifi; print(certifi.where())')
   cat /usr/local/share/ca-certificates/azcert.crt >> $AZCERTIFI
-  cat /usr/local/share/ca-certificates/azcert.crt | cmd_stdin_prefix tee -a /usr/local/lib/python3.8/site-packages/certifi/cacert.pem > /dev/null
+  cat /usr/local/share/ca-certificates/azcert.crt | cmd_stdin_prefix tee -a /usr/local/lib/3.9/site-packages/certifi/cacert.pem > /dev/null
```

Comment:
I believe this is specific to the *image* python version, which is a bit frustrating because it's decoupled from the `python_version` in the template config.  I'm not sure what the cleanest way to resolve that mismatch is yet.

Reply:
see new comment